src/utils/notifications.rs

```diff
@@ -87,7 +87,7 @@ impl<'a> Display for Notification<'a> {
             ),
             SetDefaultBufferSize(size) => write!(
                 f,
-                "defaulting to {} unpack ram",
+                "using up to {} of RAM to unpack component",
```

Comment:
I think it reads better as "to unpack component" over "for unpacking component", so that's why I went with this.

Reply:
Also, should it be singular or plural?

Reply:
It should be plural since the message is emitted once for the toolchain, not once per component.

Reply:
Okay, I think it confused me because it says it *after* it has already said `info: installing component XYZ`:

https://github.com/rust-lang/rustup/blob/d43d6c181f588b4618e930954c24b9d301d86661/tests/cli-exact.rs#L35-L38